fdb-relational-core/src/test/java/com/apple/foundationdb/relational/recordlayer/query/ExplainTests.java

```diff
@@ -67,8 +67,8 @@ public ExplainTests() {
 
     @Test
     void explainResultSetMetadataTest() throws Exception {
-        final var expectedLabels = List.of("PLAN", "PLAN_HASH", "PLAN_DOT", "PLAN_GML", "PLAN_CONTINUATION");
-        final var expectedTypes = List.of(Types.VARCHAR, Types.INTEGER, Types.VARCHAR, Types.VARCHAR, Types.STRUCT);
+        final var expectedLabels = List.of("PLAN", "PLAN_HASH", "PLAN_DOT", "PLAN_GML", "PLAN_CONTINUATION", "PLANNER_METRICS");
+        final var expectedTypes = List.of(Types.VARCHAR, Types.INTEGER, Types.VARCHAR, Types.VARCHAR, Types.STRUCT, Types.STRUCT);
```

Comment:
I'm not sure it needs to be here, but it would be nice to have a test somewhere that asserted on the values of the planner stats struct, even if it was just making sure the values weren't null or zero. Getting a more sophisticated test set of assertions is probably for another time.